Sources/Sentry/SentrySDK.m

```diff
@@ -153,14 +157,85 @@ + (void)setStartTimestamp:(NSDate *)value
     startTimestamp = value;
 }
 
+#if SENTRY_TARGET_PROFILING_SUPPORTED
+SentrySamplingContext *
+samplingContextForAppLaunches(void)
+{
+    SentryTransactionContext *transactionContext =
+        [[SentryTransactionContext alloc] initWithName:@"app.launch" operation:@"profile"];
+    transactionContext.forNextAppLaunch = YES;
+    return [[SentrySamplingContext alloc] initWithTransactionContext:transactionContext];
+}
+
+void
+configureLaunchProfiling(SentryOptions *options)
+{
+    // ???: do we need both options.enableAutoPerformanceTracing and
+    // ???: options.enableUIViewControllerTracing, or does one imply the other?
```

Comment:
Hmm, I didn't consider the SwiftUI case. I'll make that a TODO for the integration test branch. Might have to change the strategy or have a fallback for if there will be no ui.load transaction. Do you have suggestions for this?

Reply:
Still need to change the check to isTracingEnabled and enableAutoPerformanceTracing and make sure all the tests still pass. For SwiftUI, we're thinking right now we'll create a dedicated launch transaction to attach the profile to. This will be a stopgap solution until we implement continuous profiling, which will remove the requirement for presence of transactions. We also don't believe this will have significant tx billing implications.

Reply:
I wouldn't worry about SwiftUI in this PR. Let's discuss this somewhere else.

Reply:
Sounds good to me.